Dockerfile

```diff
@@ -12,10 +12,19 @@ RUN apt-get update \
         ninja-build jq \
   && rm -rf /var/lib/apt/lists/*
 
-ADD docs/requirements-doc.txt requirements.txt /
+# Normally, it is a bad idea to install rustup and cargo in
+# system directories (it should not be shared between users),
+# but this docker image is only for building packages, so I hope it is ok.
+RUN wget -q -O - https://sh.rustup.rs | \
+    RUSTUP_HOME=/usr CARGO_HOME=/usr sh -s -- -y --profile minimal --no-modify-path
 
-RUN pip3 --no-cache-dir install -r /requirements.txt \
-  && pip3 --no-cache-dir install -r /requirements-doc.txt
+ADD requirements.txt docs/requirements-doc.txt /
+ADD pyodide-build /pyodide-build
```

Comment:
This seems problematic to me -- doesn't this mean that it won't see local changes to `pyodide-build` in the source directory because we've installed this copy of pyodide-build? So if we update `pyodide-build` we would also have to update the docker image?

Reply:
I guess it is probably not different than what is already going on though. I'm a bit confused why it works.

Reply:
> I'm a bit confused why it works.

Well, the readthedocs doesn't use the docker image, it [installs from requirements file](https://github.com/pyodide/pyodide/blob/fd49100981dd09ec1c00238456baca58d513bf4c/.readthedocs.yml#L15). So it will use tot pyodide-build.

When building Pyodide, we [re-installs the pyodide-build](https://github.com/pyodide/pyodide/blob/fd49100981dd09ec1c00238456baca58d513bf4c/packages/Makefile#L16-L17), so the old pyodide-build that are bundled in the docker image will not be used anyway.

Reply:
So the only case you'll get a old documentation is when you build a documentation from a fresh docker image.